src/entry_point.py

```diff
@@ -295,6 +298,12 @@ def _conda_main():
     from conda.cli import main
 
     _fix_sys_path()
+    try:
+        no_rc = sys.argv.index("--no-rc")
```

Comment:
What's the plan for using this flag in `constructor`? Older conda-standalone versions will fail if the flag is passed 🤔  Or is it this here just for convenience and we will just set `CONDA_RESTRICT_RC_SEARCH_PATH`?

Reply:
I just submitted a draft PR: https://github.com/conda/constructor/pull/863

Essentially, I will do a version check:

```python
    if exe_name == "conda-standalone" and Version(exe_version) >= Version("24.9.0"):
        info["_ignore_condarcs_arg"] = "--no-rc"
    elif exe_name == "micromamba":
        info["_ignore_condarcs_arg"] = "--no-rc"
    else:
        info["_ignore_condarcs_arg"] = ""
```

Reply:
This will be tricky with cross-installers because we might not be able to run the to-be-bundled conda_exe, but I'll add that in the constructor PR.